src/wasm/kotlin/javatests/arcs/BUILD

```diff
@@ -16,18 +16,22 @@ filegroup(
 arcs_kt_schema(
     name = "test_schemas",
     srcs = ["schemas.arcs"],
-    package = "wasm.kotlin.tests.arcs",
+    package = "wasm.kotlin.javatests.arcs",
 )
 
-arcs_kt_library(
-    name = "test-module_lib",
-    srcs = glob(["*.kt"]),
-    visibility = ["//visibility:public"],
-    deps = [":test_schemas"],
-)
+#arcs_kt_library(
+#    name = "test-module_lib",
+#    srcs = ,
+#    visibility = ["//visibility:public"],
+#    deps = [
+#        ":test_schemas",
+#        "//third_party/kotlin/kotlin:kotlin_test",
+#        ],
+#)
```

Comment:
Why's this commented out? Just delete it?

Reply:
Thanks, I missed this.